spec/UseAtomSpec.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+use Dxw\Iguana\Extras\UseAtom;
+
+describe('UseAtom', function () {
+	// initialisation
+	beforeEach(function () {
+		$this->useAtom = new \Dxw\Iguana\Extras\UseAtom();
+
+	});
+
+	afterEach(function () {
```

Comment:
I think we probably don't need this `afterEach`, see what happens if you remove it